python/dolfinx/wrappers/io.cpp

```diff
@@ -112,6 +112,11 @@ void io(py::module& m)
            py::overload_cast<const dolfinx::function::Function<PetscScalar>&>(
                &dolfinx::io::VTKFile::write),
            py::arg("u"))
+      .def("write",
```

Comment:
Do we need a need function, or can the existing write function be extended with `t=None`?

Reply:
We need two separate functions, as they point to two separate functions in the C++ layer. Note that we should change this in a rewrite of VTKFile.